content/use-cases/use-an-escrow-as-a-smart-contract.md

```diff
@@ -0,0 +1,149 @@
+
+Let's use this scenario to help illustrate this use case: A party planner uses smart contracts to manage payments from party hosts to party vendors. Specifically, the party planner wants to use a smart contract to have the party host pay the party band 2000 XRP once they are done with their set. ***TODO: this is a far-out example -- but I had to use something really tangible to be able to make myself understand the logistics of how this might work. I'm open to any other scenario that might be more appropriate.***
+
+In this use case, the party host is the sender of the escrow, the party band is the receiver of the escrow, and the party planner is playing the role of an _oracle_. In the context of smart contracts, an oracle is a neutral third-party agent that can verify real-world events to either fulfill or invalidate a smart contract. This use case uses a human oracle for illustrative purposes, but in real-life, a software application would more likely play the role of the oracle to automate the process.
+
+Using an XRP Ledger escrow to provide this smart contract is a great arrangement because the party planner, as the third-party oracle, never "holds" the funds as one might in a traditional escrow arrangement, and can't possibly take the funds for themselves.
+
+Here’s a roadmap to the high-level tasks that these participants need to complete to use an escrow as a smart contract.
+
+
+{% set n = cycler(* range(1,99)) %}
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## Meet the prerequisites
+
+The party host (sender) must have:
+
+- An XRP Ledger [account](accounts.html#creating-accounts) that holds enough XRP to pay for escrow and any fees incurred.
+
+- Access to a [`rippled` server](install-rippled.html) that they can use to create the escrow.
+
+The party band (receiver) must have:
+
+- An XRP Ledger [account](accounts.html#creating-accounts) that can receive the XRP paid by the escrow.
+
+- The ability to look up the details of an XRP Ledger transaction hash. For example, they can use the [RPC Tool](https://developers.ripple.com/xrp-ledger-rpc-tool.html) or access a [`rippled` server](install-rippled.html) that they can use to look up the transaction hash.
+
+The party planner (oracle) must have:
+
+- The ability to generate a condition and a fulfillment.
+
+- Access to a [`rippled` server](install-rippled.html) that they can use to submit the fulfillment value and finish the escrow.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## Define the terms of the smart contract
+
+To create the escrow as a smart contract, the participants must first define the terms of the contract. In this scenario, the participants need to agree on the following details.
+
+- Should the escrow be fulfilled after a specific time has passed or after a cryptographic condition has been fulfilled?
```

Comment:
Thank you for this! Sure thing. I included it because it points out that you have these options to pick from when you are creating an escrow -- but you are right, the scenario has already picked the conditionally-held option!